catalyst_voices/apps/voices/lib/widgets/text_field/voices_date_field.dart

```diff
@@ -41,6 +42,7 @@ class _VoicesDateFieldState extends State<VoicesDateField> {
   }
 
   String get _pattern => 'dd/MM/yyyy';
+  late MaskTextInputFormatter dateFormatter;
```

Comment:
`late final` and move below  `_textEditingController` as final fields should be as the top